README.rst

```diff
@@ -19,11 +19,13 @@ Feature Stack until version 1.0
 * XEP-0199: XMPP Ping [done]
 * XEP-0363: HTTP File Upload [done]
 * XEP-0333: Chat Markers [done]
-* XEP-0045: Multi-User Chat [partial]
+* XEP-0313: Message Archive Management [done]
```

Comment:
MaM is just 'done' for fetching messages from MUCs during one was offline. There is no real MaM for every situation implemented